# Service to use the frame to help drop the objects and change a vertical grasping direction
# number of fingers defined in hand_comm.h
---
int64 ret            # Set to 1 (success) or 0 (error)
string msg           # Error description
